src/io/config.cpp

```diff
@@ -42,6 +42,19 @@ std::unordered_map<std::string, std::string> Config::Str2Map(const char* paramet
   for (auto arg : args) {
     KV2Map(&params, Common::Trim(arg).c_str());
   }
+  // define verbosity and set logging level
+  int verbosity = Config().verbosity;
+  GetInt(params, "verbose", &verbosity);
+  GetInt(params, "verbosity", &verbosity);
```

Comment:
If params have both `verbose` and `verbosity` keys I think we should still emit the following warning for this case
https://github.com/microsoft/LightGBM/blob/1cc9f9dcee15586aefdd8271a1c04ad010d3c53a/include/LightGBM/config.h#L1145-L1147
to inform user that `verbosity` has higher precedence over `verbose`.

Reply:
This is done in the `KeyAliasTransform` function after setting the verbosity and de-duplicating.